src/modules/emotes/global-emotes.js

```diff
@@ -12,6 +12,7 @@ const provider = {
     displayName: 'BetterTTV Global Emotes',
     badge: cdn.url('tags/developer.png')
 };
+const urlTemplate = '//cdn.betterttv.net/emote/{{id}}/{{image}}';
```

Comment:
we shouldn't duplicate constants. maybe put in https://github.com/night/BetterTTV/blob/master/src/utils/cdn.js